charts/jenkins/values.yaml

```diff
@@ -352,8 +352,6 @@ controller:
     defaultCrumbIssuer:
```

Comment:
I think there's no need to include this at all once we remove the client ip from crumb config

Reply:
not sure tbh. i believe not defining it would also disable the default crumb issuer right?